### DISCUSSION TITLE
Publishing status @ top center of UI to MQTT supported?

### DISCUSSION THREAD

**Discussion by rssalerno** [aqualinkd/AqualinkD#187] [Q&A] [answered]
Does aqualinkd support publishing the status message, i.e. what's shown at the top center of the UI, to MQTT?

Messages about my salt chlorine generator pop up there from time to time and I'd like use this to trigger an alert in Home Assistant.

![ui](https://user-images.githubusercontent.com/1527028/186257313-d22fdbc3-422f-4ba1-bbea-5bd7d4ace42e.jpg)

**Comment by sfeakes** [comment, marked as answer]
Yes, it will publish to `<topic>/Display_Message`, so `aqualinkd/Display_Message` with default config.   In the current release it's not documented and may have a few bugs, but it's fully functional with the release I haven't published yet.   You can pull up any MQTT browser or use mosquito_sub to see what aqualinkd does published that may not be documented.
